packages/manager/apps/web/client/app/private-database/configuration/private-database-configuration.html

```diff
@@ -6,14 +6,6 @@
     </div>
 
     <div data-ng-if="!configurationsCtrl.loading">
-        <oui-message
-            data-type="info"
-            data-ng-if="configurationsCtrl.database.type === 'postgresql'"
-        >
-            <span
-                data-translate="privateDatabase_configurations_general_postgresql_soon"
```

Comment:
You should remove also the key privateDatabase_configurations_general_postgresql_soon in translation file

Reply:
wasn't sure, thks do it right now